static/app/views/performance/trends/changeExplorerUtils/metricsTable.tsx

```diff
@@ -0,0 +1,415 @@
+      change:
+        aggregateData?.trend_percentage !== 1
+          ? formatPercentage(aggregateData?.trend_percentage! - 1, 1)
+          : '-',
+    };
+  }
+  return undefined;
+}
+
+function getEventViewWithFields(
+  _organization: Organization,
+  eventView: EventView,
+  start: string,
+  end: string,
+  fields: AggregationKeyWithAlias[],
+  eventType: string,
+  transactionName: string,
+  dataset: DiscoverDatasets
+): EventView {
+  const newEventView = eventView.clone();
+  newEventView.start = start;
+  newEventView.end = end;
+  newEventView.statsPeriod = undefined;
+  newEventView.dataset = dataset;
+  newEventView.query = 'event.type:' + eventType + ' transaction:' + transactionName;
+  newEventView.additionalConditions = new MutableSearch('');
+
+  const chartFields: QueryFieldValue[] = fields.map(field => {
+    return {
+      kind: 'function',
+      function: [field, '', undefined, undefined],
+    };
+  });
+
+  return newEventView.withColumns(chartFields);
+}
+
+function toFormattedNumber(numberString: ReactText, decimal: number) {
+  return (numberString as number).toFixed(decimal);
+}
+
+function percentChange(before: number, after: number) {
+  return (after - before) / before;
+}
+
+function renderHeadCell(column: MetricColumn, _index: number): ReactNode {
+  const align = fieldAlignment(column.key, COLUMN_TYPE[column.key]);
+  return (
+    <SortLink
+      title={column.name}
+      align={align}
+      direction={undefined}
+      canSort={false}
+      generateSortLink={() => undefined}
+    />
+  );
+}
+
+export function renderBodyCell(
+  column: GridColumnOrder<MetricColumnKey>,
+  dataRow: TableDataRow
+) {
+  let data = '';
+  let color = '';
+  if (column.key === 'change') {
+    if (
+      dataRow[column.key] === '0%' ||
+      dataRow[column.key] === '+NaN%' ||
+      dataRow[column.key] === '-'
+    ) {
+      data = '-';
+    } else if (dataRow[column.key].charAt(0) !== '-') {
+      color = theme.red300;
+      data = '+' + dataRow[column.key];
+    } else {
+      color = theme.green300;
+      data = dataRow[column.key];
+    }
+  } else {
+    data = dataRow[column.key];
+  }
+
+  return (
+    <Container data-test-id={'pce-metrics-chart-row-' + column.key}>
+      <ExplorerText
+        data-test-id={'pce-metrics-text-' + column.key}
+        align={column.key !== 'metric' ? 'right' : 'left'}
+        color={color}
+      >
+        {data}
+      </ExplorerText>
+    </Container>
+  );
+}
+
+function getQueryParams(
+  startTime: string,
+  endTime: string,
+  fields: AggregationKeyWithAlias[],
+  query: string,
+  dataset: DiscoverDatasets,
+  organization: Organization,
+  eventView: EventView,
+  transactionName: string,
+  location: Location
+) {
+  const newLocation = {
+    ...location,
+    start: startTime,
+    end: endTime,
+    statsPeriod: undefined,
+    dataset,
+    sort: undefined,
+    query: {
+      query: 'event.type:' + query + ' transaction:' + transactionName,
```

Comment:
you can use backticks so it's easier to read string's with interpolated variables